docs/components/concepts/process-instance-creation.md

```diff
@@ -67,32 +62,24 @@ When the client resends the command, it creates a new process instance.
 :::
 
 <details>
-  <summary>Code example</summary>
-  <p>
-Create a process instance and await results:
+   <summary>Create a process instance and await results via Camunda 8 REST API</summary>
+   <p>
 
 ```
-zbctl create instance "order-process" --withResult --variables '{"orderId": "1234"}'
+curl -L 'http://localhost:8080/v2/process-instances' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "processDefinitionId": "order-process”,
+  "processDefinitionVersion": 1,
+  "variables": { "orderId": "1234" }
+}'
 ```
 
-Response:
```

Comment:
❓ Like mentioned above, any reason not to show the response here as well?

Reply:
I did not yet have a chance to run the API. If it's the same, I can add it back.